product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx

```diff
@@ -36,13 +36,37 @@ You can connect to your cluster using [`psql`](http://postgresguide.com/utilitie
 
 ## Recommended settings for SSL mode
 
-Different clients can have different default TLS/SSL modes (sslmode). For example, `psql` defaults to `prefer`, which means the client will attempt to establish a TLS connection but fall back to non-TLS if the server does not support it. In the `psql` example provided by EDB in the **Quick Connect** field, `sslmode` is explicitly set to `require`, which means the client will attempt a TLS connection and fail if the connection to the server can't be encrypted.
```

Comment:
I don't think it is a bad thing to point out that sslmode is the parameter that governs the TLS/SSL modes. This was a subtle way of doing that. I would lean to keeping it in unless you think it is causing confusion.

Reply:
Added it back.